bug: 70098-70101
program: python
dataset: manybugs
dockerfile: Dockerfile.bug
build-arguments:
    scenario: "python-bug-70098-70101"
